concourse/model/traits/update_component_deps.py

```diff
@@ -60,8 +61,11 @@ class UpstreamUpdatePolicy(enum.Enum):
     ),
     AttributeSpec.optional(
         name='merge_policy',
-        default=MergePolicy.MANUAL,
+        default={
+            'manual': ['*',],
```

Comment:
- stylistic nit: there should not be a trailing comma in oneline-lists
- if this is a regex (which I hope..), this should rather read `.*`, should it not?
- I would prefer to keep the old attr (`merge_policy`), deprecate it, and introduce a new one (see comment on the PR - name it either `merge_policies`, or just `policies`; it both are present (configured), this is an error
- as a "trick", we might make the component-filter optional (absent filter meaning "match-all")

Reply:
also: if possible, always use immutable container types (tuple instead of list).